packages/react-pose/src/components/PoseElement/index.tsx

```diff
@@ -63,11 +63,11 @@ class PoseElement extends React.PureComponent<PoseElementInternalProps> {
    * =============================================
    */
   private childrenHandlers: PoseContextProps = {
-    registerChild: props => {
+    registerChild: (props: ChildRegistration) => {
```

Comment:
places like this surprise me, not sure why TS complains about this on my machine, but not on yours (and not on CI), looks like some config mismatch, but I shouldnt be using any extra config over what's in the repo here